.github/workflows/release.yml

```diff
@@ -50,3 +50,41 @@ jobs:
           tag_name: ${{ github.event.inputs.tag }}
           release_name: Release ${{ github.event.inputs.tag }}
           prerelease: ${{ github.event.inputs.prerelease }}
+
+  release-web-interface:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+
+      - name: Set up Docker Buildk
+        uses: docker/setup-buildx-action@v1
+        with:
+          buildkitd-flags: --debug
+
+      - name: Login to Registry
+        uses: docker/login-action@v1
+        with:
+          registry: registry.scality.com
+          username: ${{ secrets.REGISTRY_LOGIN }}
+          password: ${{ secrets.REGISTRY_PASSWORD }}
+
+      - name: Build and push
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          file: ./web.Dockerfile
+          push: true
+          tags: "registry.scality.com/openstack-actions-runner/web-interface:${{ github.event.inputs.tag }}"
+
+      - name: Create Release
+        uses: softprops/action-gh-release@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          tag_name: ${{ github.event.inputs.tag }}
+          release_name: Release ${{ github.event.inputs.tag }}
+          prerelease: ${{ github.event.inputs.prerelease }}
```

Comment:
What I don't like here (and I don't even know if it will work) is that you're trying to tag the repository twice.

Here what I suggest you to do:
* Split the workflow in three jobs.
* Two jobs that will be executed at the same time and build each docker image and push it to the registry.
* A third job that will just create the GitHub release. and "needs" the other two job to finish before starting.

Or you can also consider adding everything in the same job and just execute a docker build after the other

Reply:
![image](https://user-images.githubusercontent.com/22933068/136168719-c9af2bc3-8b27-4eed-b0ff-d944e2280e5b.png)
:+1: